### ISSUE TITLE
Discovery TXT record contents

### ISSUE THREAD

**Issue by TheEnbyperor** [Domain-Connect/spec#103]
It is currently unclear in the specification if the contents of the `_domainconnect` TXT record is a FQDN, or a partial URI. That is can the record contain a path element; `example.net/connect` being valid as well as `example.net`.

From poking at iCloud's implementation they seem to treat it as partial URI and not an FQDN. I'm unsure what other current implementations do. 

As for my opinion on this I think it should be a partial URI, and not just an FQDN.

**Comment by pawel-kow** [comment]
Excellent question!
The specification is obviously not specific enough about it. I would rather opt for `partial URI` interpretation - this is at least what can be derived from the text how the URL to `/settings` REST API is built.
The fact that the spec is now silent about it and all examples actually use FQDN may be indeed misleading.

**Comment by greenreader9** [comment]
I agree with @TheEnbyperor that a partial URI should be considered for the spec over FQDN. 

A template such as <scheme>://<hostname>/<optionalPath> could be provided in the spec sheet at well. 

I do not think that queries or fragments should be allowed.

**Comment by greenreader9** [comment]
I agree with @TheEnbyperor, in that a FQDN should not be required, but a partial URI should be allowed. 

Since the service provider just makes a request to the endpoint suffixed with the required paths and parameters, the DNS provider should be allowed to do their own versioning within paths (such as "dnsprovider.tld/api/v1/domainConnect/". 

As long as a valid URI is provided, I don't see a reason why paths should not be allowed.
